doc/overview/security.md

```diff
@@ -1,14 +1,61 @@
 <a id="4.4"></a>
 # Security Model
 
-DAOS uses a flexible security model that seperates authentication from authorization. It is designed to have very minimal impact on the I/O path.
+DAOS uses a flexible security model that seperates authentication from
+authorization. It is designed to have a minimal impact on the I/O path.
+
+There are two areas of DAOS that require access control. At the user level,
+clients must be able to read and modify only pools and containers to which they
+have been granted access. At the system and administrative levels, only
+authorized components must be able to access the DAOS management network.
 
 <a id="4.4.1"></a>
 ## Authentication
 
-The DAOS security model is designed to support different authentication methods. By default, a local agent runs on the client node and authenticats the user process through AUTH_SYS. Authentication can be handle by a third party service like munge or Kerberos.
+There are different means of authentication depending on whether the caller is
+accessing client resources or the DAOS management network.
+
+### Client Library
+
+The client library is an untrusted component. A trusted process, the DAOS agent,
+runs on the client node and authenticates the user process.
+
+The DAOS security model is designed to support different authentication methods
+for client processes. Currently, we support AUTH_SYS authentication only.
+
+### DAOS Management Network
+
+Each trusted DAOS component (agent, server, and administrative tool) is
+authenticated by means of a certificate generated for that component. These
+components identify one another over the DAOS management network via
+mutually-authenticated TLS.
 
 <a id="4.4.2"></a>
 ## Authorization
 
-DAOS supports a subset of the NFSv4 ACLs for both pools and containers through the properties API.
+Client authorization for resources is controlled by the Access Control List
+(ACL) on the resource, while authorization on the mamagement network is
+achieved by settings on the certificates generated while setting up the DAOS
```

Comment:
Worth making this a hyperlink to the section on generating certs?

Reply:
Good idea, I added a link to the deployment doc section about certificate configuration.